tests/InputFiles/Burgers/Step.yaml

```diff
@@ -21,18 +21,20 @@ DomainCreator:
   Interval:
     LowerBound: [-1.0]
     UpperBound: [1.0]
-    IsPeriodicIn: [false]
     InitialRefinement: [2]
     InitialGridPoints: [7]
     TimeDependence: None
+    BoundaryConditions:
+      LowerBoundary: DirichletAnalytic
+      UpperBoundary: DirichletAnalytic
 
 SpatialDiscretization:
+  BoundaryCorrection:
+    Hll:
```

Comment:
just checking if it was intentional to switch to HLL

Reply:
Yea, it's what people should be using, I think :)